tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts

```diff
@@ -50,7 +50,7 @@ suite(`${stack} test`, async () => {
     });
 
     suite('Validation of workspace build', async () => {
-        codeExecutionTests.runTask(buildTaskName, 420_000);
+        codeExecutionTests.runTask(buildTaskName, 1420_000);
```

Comment:
With your manual testing, did the build really last over 20 minutes?

Reply:
Last time it took 18 minutes.

Reply:
AOK